setup.py

```diff
@@ -8,6 +8,9 @@
 import os
 import platform
 import shutil
+from numpy import array
```

Comment:
lgtm but inefficient

Import entire modules instead of individual symbols within a module. Do this!

Exception: For third-party code where documentation explicitly says to import individual symbols.

Put all imports at the top of the page ...

📄 [Steven Loria](https://gist.github.com/sloria/7001839#imports)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
test 3/23/2022

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Secure, Inefficient, Fault-tolerant

Reply:
test smoke 23

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Elegant, Not reusable, Maintainable

Reply:
test

Reply:
no snippets test __
-tolerant
__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Readable, Inefficient, Fault-tolerant